app/src/main/java/woowacourse/shopping/feature/main/MainActivity.kt

```diff
@@ -100,20 +133,30 @@ class MainActivity : AppCompatActivity(), MainContract.View {
         startActivity(DetailActivity.getIntent(this, recentProduct.productUiModel))
     }
 
-    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
-        menuInflater.inflate(R.menu.app_bar_menu, menu)
-        return true
+    override fun updateCartProductCount(count: Int) {
+        if (::cartProductCountTv.isInitialized) {
+            if (count == 0) cartProductCountTv.visibility = View.GONE
+            else cartProductCountTv.visibility = View.VISIBLE
```

Comment:
아래처럼 간단하게 표현할수도 있을거같아요! 
```suggestion
            cartProductCountTv.isVisible = count != 0
```